server/webhook_parser.go

```diff
@@ -162,6 +162,8 @@ func parseWebhookChangeLog(jwh *JiraWebhook) Webhook {
 			event = parseWebhookUpdatedField(jwh, eventUpdatedAffectsVersion, field, fieldId, fromWithDefault, toWithDefault)
 		case field == "reporter":
 			event = parseWebhookUpdatedField(jwh, eventUpdatedReporter, field, fieldId, fromWithDefault, toWithDefault)
+		case field == "Component":
```

Comment:
Are the capitalization of the webhook fields actually this random?

Reply:
Yes, at least they are consistent across cloud and server!

Reply:
Looking back, maybe it would have been better to convert the capitalization of the string at the beginning of the function for cleaner code.